features/step_definitions/assertion_steps.rb

```diff
@@ -0,0 +1,8 @@
+  
+Then("I should see {string}") do |content|
+    expect(page).to have_content content
+  end
+
+Then("I should be on the {string} page") do |content|
```

Comment:
Why do you have this step def?

Reply:
Originally I was going to build separate pages for each link just to make sure the links were wired correctly. . .and I made the step def for it.  But then decided that was out of scope and we just needed navigation so I trimmed it out. I'll remove this until we use it.

Fixed!